app/main/views.py

```diff
@@ -517,3 +520,116 @@ def mousereport_pdf(animal_id):
     stylesheets = [CSS(url_for('static', filename='styles.css')),
                    CSS(url_for('static', filename='datajoint.css'))]
     return render_pdf(HTML(string=html), stylesheets=stylesheets)
+
+
+@main.route('/surgery', methods=['GET', 'POST'])
+def surgery():
+    fexperiment = dj.create_virtual_module("csmith_testing", "csmith_testing")
+    form = forms.SurgeryForm(request.form)
+    if request.method == 'POST' and form.validate():
+        tuple_ = {'animal_id': form['animal_id'].data, 'date': str(form['date'].data),
+                  'time': str(form['time_input'].data), 'username': form['user'].data,
+                  'surgery_outcome': form['outcome'].data, 'surgery_quality': form['surgery_quality'].data,
+                  'surgery_type': form['surgery_type'].data, 'weight': form['weight'].data,
+                  'ketoprofen': form['ketoprofen'].data, 'notes': form['notes'].data}
+        status_tuple_ = {'animal_id': tuple_['animal_id'], 'date': tuple_['date'], 'checkup_notes': ''}
+        if tuple_['ketoprofen'] == 0:
+            tuple_.pop('ketoprofen')
+        if tuple_['weight'] == 0:
+            tuple_.pop('weight')
+        if not fexperiment.Surgery().proj() & tuple_:
+            try:
+                fexperiment.Surgery().insert1(tuple_)
+                fexperiment.SurgeryStatus.insert1(status_tuple_)
```

Comment:
I thought you weren't gonna insert a status when the surgery is first registered but only the checkups